test/metabase/query_processor_test.clj

```diff
@@ -154,7 +156,9 @@
      :id          {:special_type :type/PK
                    :base_type    (data/id-field-type)
                    :name         (data/format-name "id")
-                   :display_name "ID"}
+                   :display_name "ID"
+                   :min-value    nil
```

Comment:
This seems wrong to me. From what I can tell, dashes are converted to underscores before being stored, then converted back once it comes out of the database. Not sure why `min-value` is dashes here instead of underscores.